integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature

```diff
@@ -279,6 +290,32 @@ Feature: PublicLightingScheduleManagement Set Light Schedule
       | Protocol    | WeekDay     | StartDay   | EndDay     | ScheduledTime | ActionTime   | Time         | TriggerWindow | LightValues | TriggerType   |
       | OSLP ELSTER | ABSOLUTEDAY | 2016-01-01 | 2016-12-31 | 2016-12-15    | ABSOLUTETIME | 18:00:00.000 |         30,30 | 0,true,     | LIGHT_TRIGGER |
 
+  Scenario Outline: Set light schedule with 2 schedules containing illegal combinations of ActionTime, TriggerWindow and TriggerType # Fail
+    Given an ssld oslp device
+      | DeviceIdentification | TEST1024000000001 |
+      | Protocol             | <Protocol>        |
+    When receiving a set light schedule request for 2 schedules
+      | DeviceIdentification | TEST1024000000001 |
+      | WeekDay              | <WeekDay>         |
+      | StartDay             | <StartDay>        |
+      | EndDay               | <EndDay>          |
+      | ActionTime           | <ActionTime>      |
+      | Time                 | <Time>            |
+      | LightValues          | <LightValues>     |
+      | TriggerType          | <TriggerType>     |
+      | TriggerWindow        | <TriggerWindow>   |
+    And the platform buffers a set light schedule response message for device "TEST1024000000001" that contains a soap fault
+      | FaultCode   | SOAP-ENV:Client  |
+      | FaultString | Validation error |
+
+    Examples: 
+      | Protocol    | WeekDay | StartDay | EndDay | ActionTime   | Time         | TriggerWindow | LightValues | TriggerType   |
+      | OSLP ELSTER | ALL     |          |        | ABSOLUTETIME | 18:00:00.000 |               | 2,true,     | ASTRONOMICAL  |
+      | OSLP ELSTER | ALL     |          |        | SUNRISE      |              |               | 2,true,     | LIGHT_TRIGGER |
+      | OSLP ELSTER | ALL     |          |        | SUNSET       |              |               | 2,true,     | LIGHT_TRIGGER |
```

Comment:
Should we also validate on triggerwindow 0,0 for a light trigger? Or define a minimum range? Or define a minimum minutes before and after the calculated sunset/sunrise? Perhaps something to discuss with product owner?

Reply:
@smvdheijden I think it's a good idea to have an extended set of validations, including all limit values. Additional test with a trigger window 0,0 and with negative and positive value are valuable in my opinion.